cli/cmd/docker.go

```diff
@@ -44,6 +47,11 @@ func (c *ServicedCli) initDocker() {
 					cli.StringFlag{"registry", "", "host:port where the registry is running"},
 					cli.BoolFlag{"override, f", "overrides all existing image records"},
 				},
+			}, {
+				Name:        "override",
+				Usage:       "Replace an image in the registry with a new image",
+				Description: "serviced docker override NEWIMAGE OLDIMAGE",
```

Comment:
To match typical Unix command style, should we switch this to "OLDIMAGE NEWIMAGE". cp, mv, ln, even diff kind of, all use this pattern.